pbm/backup/delete.go

```diff
@@ -321,17 +322,23 @@ func DeleteBackupBefore(
 	conn connect.Client,
 	t time.Time,
 	bcpType defs.BackupType,
+	profile string,
```

Comment:
Did you consider to inject here `storage.Storage` instead `profile`? That at least would be more testable, but not sure does it require significant refactoring.

Reply:
I wanted to do the least invasive change, but that was last week. Now I feel a bit more comfortable in this part of code base so I will change it. Thanks for the suggestion.